src/components/navigation/navbar/Navbar.vue

```diff
@@ -1,50 +1,45 @@
 <template>
     <div v-if="role === Role.ADMIN">
-        <admin-navbar-desktop class="hidden md:flex" />
-        <admin-navbar-mobile class="flex md:hidden" />
+        <admin-navbar-desktop v-if="width >= 768" class="flex" />
```

Comment:
should set a breakpoint in code `const breakpoint = 768` for better maintainability